src/representation/promp/manifest.xml

```diff
@@ -5,7 +5,7 @@
   <depend package="learning/bolero/include" />
   <depend package="learning/bolero/src/bl_loader" />
   <depend package="simulation/lib_manager" />
-  <depend package="external/yaml-cpp" />
+  <depend package="yaml-cpp" />
```

Comment:
Are you sure that this works on all supported platforms? Why didn't pybob complain about this?

Reply:
The system yaml-cpp is supported everywhere. pybob has nothing to with it, also autoproj can use both. But a problem occurs when some applications require the system package (as in many rock application) and some the external, because it may happen that for building and linking different libraries are used because version differences are not detected.